src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -683,8 +702,8 @@ def forward(
         >>> container = av.open(file_path)
 
         >>> # sample 32 frames
-        >>> indices = sample_frame_indices(clip_len=32, frame_sample_rate=1, seg_len=len(videoreader))
-        >>> video = videoreader.get_batch(indices).asnumpy()
+        >>> indices = sample_frame_indices(clip_len=32, frame_sample_rate=4, seg_len=container.streams.video[0].frames)
+        >>> video = read_video_pyav(container=container, indices=indices)
```

Comment:
When `seg_len` is set to the whole video length `frame_sample_rate` needs to be set to `4` otherwise the model will assign a wrong class to the video invalidating the whole example.